File: kattis/easy/java/avion.java
package easy;

import java.util.ArrayList;
import java.util.Scanner;

public class avion {
	public static void main(String[] args) {
		try (Scanner scanner = new Scanner(System.in)) {

			ArrayList<Integer> lines = new ArrayList<>();

			for (int i=0; i < 5; i++) {
				String line = scanner.nextLine();
				if (line.toLowerCase().contains("fbi"))
					lines.add(i+1);
			}

			if (lines.size() == 0) System.out.println("HE GOT AWAY!");
			else lines.forEach(s -> System.out.print(s + " "));
		}
	}
}

File: kattis/easy/cpp/avion.cpp
//
// Created by BBJ on 02.01.2024.
//
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

int main() {
    std::vector<int> lines;

    for (int i=0; i  < 5; ++i) {
        std::string line; std::cin >> line;

        // string.tolower
        std::transform(line.begin(), line.end(), line.begin(), ::tolower);

        // fbi-check
        if (line.find("fbi") != std::string::npos)
            lines.push_back(i+1);
    }
    if (lines.empty()) std::cout << "HE GOT AWAY!";
    else for (const auto& num : lines) std::cout << num << " ";
    return 0;
}